### DISCUSSION TITLE
Macintosh Plus VNC Server Implementation

### DISCUSSION THREAD

**Discussion by akuker** [akuker/RASCSI#651] [Show and tell]
@marciot posted this in Issue #649 , so I thought I'd move it over to the discussions area

This is demonstration of a project I have been working on since last year that makes use of the RaSCSI. It is an experimental remote desktop server for classic Macintosh computers:

Mini VNC Demonstration

[YouTube](https://youtu.be/zM_sNItbuhc)

To my knowledge, it is the first VNC server that can run on a Macintosh computer with a 68000 processor (there was at one point something called [ChromiVNC](https://web.archive.org/web/20070208223046/http://www.chromatix.uklinux.net/vnc/index.html) but that required Open Transport and at least a 68030 CPU).

Anyhow, this project is pretty much a work-in-progress and it's not tremendously stable or usable for real work. Not sure whether it's my code or whether MacTCP is inherently unstable. Does anyone here have any development experience with MacTCP and can tell me whether they've had problems with it crashing under load?

Anyhow, if you want to learn more about how I accomplished this, please subscribe to my channel. I had to rely on a lot of technical hacks to get this to work and I've definitely gone down some very deep rabbit holes.

If there is interest, I might post a video with a more behind-the-scenes look at this project and the coding hacks I had to do to make this work 😁

**Comment by marciot** [comment]
There is now also a thread on [68kmla](https://68kmla.org/bb/index.php?threads/experimental-vnc-remote-desktop-server-for-the-classic-macintosh.40096/page-2)
